### ISSUE TITLE
Лидирующие пробелы многострочных параметров

### ISSUE THREAD

**Issue by lintest** [Pr-Mex/vanessa-automation#1355]
При обработке многострочных параметров сейчас теряются лидирующие пробелы. Из-за потери лидирующих пробелов многострочные параметры сейчас нельзя использовать для формирования Markdown и в некоторых других случаях

**Comment by Pr-Mex** [comment]
Да, проблема есть. Есть обход - передавать в таком случае в шаг таблицу Gherkin.
|'     текст     '|
